Dockerfile

```diff
@@ -108,3 +108,8 @@ RUN wget -nv https://github.com/balhoff/relation-graph/releases/download/v$RG/re
 && tar -zxvf relation-graph-$RG.tgz \
 && mv relation-graph-$RG /tools/relation-graph \
 && chmod +x /tools/relation-graph
+
+# rdftab
+ENV RDFTAB=0.1.1
+RUN wget -nv https://github.com/ontodev/rdftab.rs/releases/download/v$(RSDTAB)/rdftab-x86_64-unknown-linux-musl -O /tools/rdftab \
```

Comment:
This won’t work on M1-powered Macs.

There doesn’t seem to be [pre-built binaries](https://github.com/ontodev/rdftab.rs/releases/tag/v0.1.1) for rdftab targeting the arm64 architecture, so for M1 we will (again) need to build from source. Luckily we already have Rust development tools on the `odkbuilder` image (we need them to build Fastobo).

Reply:
We would be open to building and releasing an arm64 M1. Would it be as simple as adding to our GitHub Action here? https://github.com/ontodev/rdftab.rs/blob/master/.github/workflows/rust.yml#L31

Reply:
Cargo seems to have built-in support for cross-compiling, so it could indeed be as simple as calling `cargo build --target aarch64-unknown-linux-musl`.

Reply:
@gouttegd Can you slowly slowly sheperd this PR through to completion? If not, let me know and I will try to find some time. Thank you!

Reply:
@matentzn On it. :) I am currently testing building `rdftab` from source from within the `odkbuild` image (same approach as the one we are already using for `fastobo`). I’ll push an amendment to this PR as soon as the test is conclusive.